library/tinkerpop/src/test/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraphTest.java

```diff
@@ -69,6 +67,67 @@ public class GafferPopGraphTest {
         }
     };
 
+    private static final Configuration TEST_CONFIGURATION_2 = new BaseConfiguration() {
+        {
+            this.setProperty(GafferPopGraph.OP_OPTIONS, new String[] {"key1:value1", "key2:value2" });
+            this.setProperty(GafferPopGraph.USER_ID, USER_ID);
+            this.setProperty(GafferPopGraph.DATA_AUTHS, new String[]{AUTH_1, AUTH_2});
+            this.setProperty(GafferPopGraph.GRAPH_ID, "Graph1");
+            this.setProperty(GafferPopGraph.STORE_PROPERTIES, GafferPopGraphTest.class.getClassLoader().getResource("gaffer/store.properties").getPath().toString());
```

Comment:
It's already a String.
```suggestion
            this.setProperty(GafferPopGraph.STORE_PROPERTIES, GafferPopGraphTest.class.getClassLoader().getResource("gaffer/store.properties").getPath());
```